src/Pact/Eval.hs

```diff
@@ -398,11 +543,16 @@ resolveFreeVars i b = traverse r b where
              Nothing -> evalError i $ "Cannot resolve " ++ show fv
              Just d -> return d
 
+-- Install must handle interfaces
+-- strip out everything except consts
```

Comment:
This comment seems inaccurate, and prob. unnec

Reply:
Leftover - good catch